Analysis/DEEP2_R23_O32/Plotting/te_metal_plots.py

```diff
@@ -18,20 +18,56 @@
 from Metallicity_Stack_Commons.column_names import filename_dict
 
 jiang_coeff = [-24.135, 6.1532, -0.37866, -0.147, -7.071]
+bian_coeff = [138.0430, -54.8284, 7.2954, -0.32293]
 
-a = -24.135
-b = 6.1532
-c = -0.37866
-d = -0.147
-e = -7.071
 
+def jiang_calibration(metal_det, lo32):
+    """
+    Purpose
+    Calculating the Jiang calibration for the metallicity and O32 measurements enter
```

Comment:
Provide the full reference for Jiang et al.

Reply:
fixed